lambdas/indexer/app.py

```diff
@@ -137,3 +147,11 @@ def aggregate_retry(event: chalice.app.SQSEvent):
                     batch_size=1)
 def contribute_retry(event: chalice.app.SQSEvent):
     app.index_controller.contribute(event, retry=True)
+
+
+if not app.unit_test and config.deployment_stage == aws.shared_deployment_stage:
```

Comment:
Would be appropriate to explain the need for `not app.unit_test`. What breaks if it is absent?

Reply:
The other aspect of the conditional might also need a short explanation.